server/routes/changeTeam/handlers/changeTeam.ts

```diff
@@ -2,7 +2,12 @@ import { RequestHandler, Request, Response } from 'express'
 import CaseloadService from '../../../services/caseloadService'
 
 export default class ChangeTeamRoutes {
-  constructor(private readonly caseloadService: CaseloadService) {}
+  constructor(
+    private readonly caseloadService: CaseloadService,
+    private section: string
```

Comment:
little trick - you can do `private section: string` and you don't need to assign the variable in a constructor

you can refer to it as  this.section regardless